internal/common/common.go

```diff
@@ -42,6 +42,13 @@ const (
 	RoleStandby   Role = "standby"
 )
 
+// Roles enumerates all possible Role values
```

Comment:
This list needs maintaining to ensure we can set all possible label values to 0 on stolon_keeper_{local,target}_role before setting our active metric to be 1. This becomes clearer when you see the output of a single instances metrics:

![image](https://user-images.githubusercontent.com/3518874/57087886-fd5e0f80-6cf8-11e9-8748-e5e14f79ff42.png)